src/components/accessories/admission/consts.ts

```diff
@@ -12,7 +12,7 @@ export const initialFields: TFields<AdmissionFormFieldName> = {
     type: "number",
   },
   admDate: {
-    value: "",
+    value: new Date(Date.now()).toISOString(),
```

Comment:
the toISOString methond doesn't take care about the timezone, which could cause to some incongruences with the date. Please see what I do inside the libraries/formDataHandling/functions.ts:29 . Maybe we can extract the code at 32 - 35 and export it inside another method which we can use to get the right date inside the rest of the code.